central/localscanner/certificates.go

```diff
@@ -16,27 +18,45 @@ func IssueLocalScannerCerts(namespace string, clusterID string) (*storage.TypedS
 	if namespace == "" {
 		return nil, errors.New("namespace is required to issue the certificates for the local scanner")
 	}
-
+	var certs = map[storage.ServiceType]*storage.TypedServiceCertificate{
```

Comment:
I think we can just do a `slice` of `storage.ServiceType` and have another `slice` of `*storage.TypedServiceCertificate` and only `append` to it when `localScannerCertificatesFor` does not return an `error`. This way, we just have the single loop.

Reply:
Thanks, I have rewritten the code.